[2025-08-26T08:53:44.765+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T08:51:28.137970+00:00 [queued]>
[2025-08-26T08:53:44.774+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T08:51:28.137970+00:00 [queued]>
[2025-08-26T08:53:44.775+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-26T08:53:44.788+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): cleanup_temp_files> on 2025-08-26 08:51:28.137970+00:00
[2025-08-26T08:53:44.794+0000] {standard_task_runner.py:60} INFO - Started process 129 to run task
[2025-08-26T08:53:44.796+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mamda_monthly_etl_pipeline', 'cleanup_temp_files', 'manual__2025-08-26T08:51:28.137970+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/mamda_monthly_etl.py', '--cfg-path', '/tmp/tmpv2emllfi']
[2025-08-26T08:53:44.798+0000] {standard_task_runner.py:88} INFO - Job 115: Subtask cleanup_temp_files
[2025-08-26T08:53:44.847+0000] {task_command.py:423} INFO - Running <TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T08:51:28.137970+00:00 [running]> on host 9f5b6f3413d6
[2025-08-26T08:53:44.930+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mamda_etl' AIRFLOW_CTX_DAG_ID='mamda_monthly_etl_pipeline' AIRFLOW_CTX_TASK_ID='cleanup_temp_files' AIRFLOW_CTX_EXECUTION_DATE='2025-08-26T08:51:28.137970+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-26T08:51:28.137970+00:00'
[2025-08-26T08:53:44.932+0000] {mamda_monthly_etl.py:203} INFO - Cleanup finished
[2025-08-26T08:53:44.932+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-26T08:53:44.943+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=mamda_monthly_etl_pipeline, task_id=cleanup_temp_files, execution_date=20250826T085128, start_date=20250826T085344, end_date=20250826T085344
[2025-08-26T08:53:44.970+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-26T08:53:44.985+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
